content/en/blog/2023/why-histograms/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Why Histograms?
+date: 2023-05-05
+author: '[Daniel Dyla](https://github.com/dyladan)'
+---
+
+<i>Originally posted to
+https://dyladan.me/histograms/2023/05/02/why-histograms/</i>
+
+A histogram is a multi-value counter which summarizes the distribution of one or
+more data points. For example, a histogram may have 3 counters which count the
+occurrences of negative, positive, and zero values respectively. Given a series
+of numbers, `3`, `-9`, `7`, `6`, `0`, and `-1`, the histogram would count `2`
+negative, `1` zero, and `3` positive values. A single histogram data point is
+most commonly represented as a bar chart.
+
+![histogram point as bar chart](hist-point.png "A single histogram point plotted as a bar chart with 3 buckets titled 'Positivity of numbers'. The first bucket shows negative numbers and has a height of 2. The second bucket shows zero values and has a height of 1. The third bucket shows positive values and has a height of 3.")
+
+This simple example has only 3 possible output values, but it is not uncommon to
+have many more in a single histogram. Similarly, this shows only 1 histogram
+data point, but in the world of observability data is usually constantly
+exported. For example, an application may export a histogram every minute which
+summarizes a metric for the previous minute. In this way, you can study how the
+distribution of your data changes over time.
+
+# What are histograms for?
```

Comment:
```suggestion
## What are histograms for?
```

Reply:
Every heading is H2? Is that a style thing for the Otel blog in general?